src/lang/english.txt

```diff
@@ -5128,3 +5130,36 @@ STR_PLANE                                                       :{BLACK}{PLANE}
 STR_SHIP                                                        :{BLACK}{SHIP}
 
 STR_TOOLBAR_RAILTYPE_VELOCITY                                   :{STRING} ({VELOCITY})
+
+STR_BUILD_VEHICLE_SMART                                         :Smart build vehicle menu
+STR_BUILD_VEHICLE_SMART_HELPTEXT                                :Smart build vehicle menu
+STR_BUILD_VEHICLE_DETAIL                                        :{BLACK}Detail
+STR_BUILD_VEHICLE_DETAIL_TOOLTIP                                :{BLACK}Extended options for comfortable unit select
+STR_BUILD_VEHICLE_AUTO_TRAIN                                    :{BLACK}Take the train parameters from the from
+STR_BUILD_VEHICLE_AUTO_TRAIN_TOOLTIP                            :{BLACK}Take parameters from the first train from the depot (maximum weight, available speed, train length)
+STR_BUILD_VEHICLE_FILTER                                        :{BLACK}Use filters...
```

Comment:
OTTD generally does not use ellipses to indicate whether a button/menu will open a new window. Just leave it out.

Reply:
Unfortunately, I do not understand what you mean. Button shows / hides the panel with additional filters. The point is to see and adjust filters only if necessary. In a vanilla game, for example, this is usually not necessary.